Source/VCGeneration/Splits/SplitAttributeHandler.cs

```diff
@@ -0,0 +1,216 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using Microsoft.Boogie;
+using VC;
+
+namespace VCGeneration;
+
+class SplitAttributeHandler {
```

Comment:
Is this file only refactored from somewhere else?

Reply:
It's part of the old ManualSplitFinder.cs, but it's changed quite a bit.